packages/forms-web-app/src/dynamic-forms/dynamic-components/listed-building-add-more/question.js

```diff
@@ -0,0 +1,61 @@
+const AddMoreQuestion = require('../add-more/question');
+
+const uuid = require('uuid');
+
+class ListedBuildingAddMoreQuestion extends AddMoreQuestion {
+	constructor({
+		title,
+		question,
+		fieldName,
+		viewFolder,
+		html,
+		validators,
+		label,
+		inputClasses = 'govuk-input--width-10'
+	}) {
+		super({
+			title,
+			viewFolder,
+			fieldName,
+			question,
+			html,
+			validators
+		});
+
+		this.inputClasses = inputClasses;
+		this.label = label;
```

Comment:
The UI label is managed on the parent ListAddMoreQuestion class here https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/main/packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js#L33

Reply:
Yep, however currently that label is only used for displaying the label to use in the list.
I tried changing it so it also displays the sub question label before the input field however that also affects the existing list add more questions in which we do not expect that label to show. 
For instance on the screenshot below the word 'appeal' is not expected to show before the input.
<img width="907" alt="Screenshot 2023-10-10 at 11 45 47" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/16866068/c1129a52-43ed-403f-bcfd-4e665209125f">